roles/lib_utils/filter_plugins/openshift_master.py

```diff
@@ -454,13 +454,17 @@ class GitHubIdentityProvider(IdentityProviderOauthBase):
     def __init__(self, api_version, idp):
         IdentityProviderOauthBase.__init__(self, api_version, idp)
         self._optional += [['organizations'],
-                           ['teams']]
+                           ['teams'],
+                           ['ca'],
+                           ['hostname']]
 
     def validate(self):
         ''' validate this idp instance '''
         if self.challenge:
             raise errors.AnsibleFilterError("|failed provider {0} does not "
                                             "allow challenge authentication".format(self.__class__.__name__))
+        if self._idp['ca'] != "":
```

Comment:
I don't think it needs to be conditional. See other providers implementation, https://github.com/openshift/openshift-ansible/issues/9397#issuecomment-413158710 and https://bugzilla.redhat.com/show_bug.cgi?id=1633137

Reply:
I can remove it, do you think there should be a conditional checking if the file is there/exists? If not the installation will fail (when masters get restarted).

Reply:
If the admin doesn't use `_ca` or `_ca_file` vars then its their responsibility to have this file present there - a different playbook, host image etc.